lib/components/molecules/QuoteMedia/types/index.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  className?: string,
+  title?: string,
```

Comment:
here you have title as optional but in your component thats not the case

Reply:
fixed